### PR TITLE
Add RDM package "aru.Cichy2019" 

### PR DESCRIPTION
**Pull request by hermessuen**

### CODE REVIEW

**Review by mschrimpf — APPROVED** [review]

**Comment by mschrimpf** [comment]
@hermessuen it seems like the test failed when the loader was trying to add the StimulusSet metadata to the assembly by joining on the `image_id` coordinate: `KeyError: 'image_id'` (http://braintree.mit.edu:8080/job/unittest_brainio/185/console).
Does the assembly not have an `image_id`?

**Comment by hermessuen** [comment]
@mschrimpf  Sorry for the later response on this - the RDM does not have an image_id and  is presentation x presentation x subject. What would be the best way to add it?

**Comment by mschrimpf** [comment]
The way the system is setup now, your assembly is required to have an `image_id` coordinate (when you have a `presentation` dimension). It is supposed to uniquely identify the image so that metadata from the stimulus_set can be [added](https://github.com/brain-score/brainio_collection/blob/abe1a034a51cc2c5f422bd4348c41121cc45d6e3/brainio_collection/fetch.py#L125) to the assembly. You can add it like any other coordinate, see e.g. [here](https://github.com/brain-score/brainio_contrib/blob/fab98ee2ec50a14f9f73f7d6231842cd4b229fd3/mkgu_packaging/dicarlo/sanghavi/sanghavi2020.py#L42)

**Comment by hermessuen** [comment]
I added a coordinate, re-uploaded, and altered the lookup.csv in my repo!

File: brainio_collection/lookup.csv [hunk 1]
```diff
@@ -226,6 +226,10 @@ dicarlo.Seibert2019,assembly,NeuronRecordingAssembly,S3,https://brainio.dicarlo.
 aru.Kuzovkin2018,stimulus_set,StimulusSet,S3,https://brainio.contrib.s3.amazonaws.com/image_aru_Kuzovkin2018.csv,a5990b24aea3e453756141cbe69a83304db72d0b,
 aru.Kuzovkin2018,stimulus_set,,S3,https://brainio.contrib.s3.amazonaws.com/image_aru_Kuzovkin2018.zip,cca4d819d7743bdd4bf65c1cb2439fd0ec97543a,
 aru.Kuzovkin2018,assembly,NeuronRecordingAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_aru_Kuzovkin2018.nc,5fae8b283a043562ce9925d48ad99db151f39067,aru.Kuzovkin2018
+aru.Cichy2019,stimulus_set,StimulusSet,S3,https://brainio.contrib.s3.amazonaws.com/image_aru_Cichy2019.csv,851dcae666ab9a555e3a60371c02f1d433f91077,
+aru.Cichy2019,stimulus_set,,S3,https://brainio.contrib.s3.amazonaws.com/image_aru_Cichy2019.zip,d2166dd9c2720cb24bc520f5041e6830779c0240,
+aru.Cichy2019,assembly,DataAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_aru_Cichy2019.nc,81238060ca1faa5d1875fd2b6321fd7763578d35,aru.Cichy2019
+aru.Cichy2019,assembly,DataAssembly,S3,https://brainio.contrib.s3.amazonaws.com/assy_aru_Cichy2019.nc,9224de46b32b61e6488f68171214c7d42b99db02,aru.Cichy2019
```
Comment (mschrimpf) [inline brainio_collection/lookup.csv:232]: looks good, but make sure to delete the previous lookup line for the assembly

Comment (hermessuen) [reply brainio_collection/lookup.csv]: Oops! thought I had deleted. Just re-submitted

**Pull request merged**